addons/account/views/account_move_views.xml

```diff
@@ -837,6 +837,22 @@
                                             'show_address': 1, 'default_is_company': True, 'show_vat': True}"
                                        options='{"always_reload": True, "no_quick_create": True}'
                                        attrs="{'invisible': [('move_type', 'not in', ('out_invoice', 'out_refund', 'in_invoice', 'in_refund', 'out_receipt', 'in_receipt'))]}"/>
+                                <label for="journal_id"
+                                       groups="account.group_account_readonly"
+                                       invisible="context.get('default_journal_id') and context.get('move_type', 'entry') != 'entry'"/>
```

Comment:
Not sure why this depends on the context?
It means that the view is displayed differently depending on where you come from, and not depending on the document itself?

Reply:
It was originally like this (the code is just moved around here, for faster "tab" navigation) so I tried to minimize the diff